sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -367,85 +219,65 @@ public void UpsertFxRate(string scope, DateTimeOffset effectiveFrom,  DateTimeOf
+                    var inverseFxRate = BuildSimpleQuoteUpsertRequest("JPY/USD", QuoteSeriesId.InstrumentIdTypeEnum.CurrencyPair, 1m / 150, "USD", date);
 
                     upsertQuoteRequests.Add($"day_{days}_fx_rate", fxRate);
                     upsertQuoteRequests.Add($"day_{days}_inverseFx_rate", inverseFxRate);
                 }
                 else
                 {
-                    var fxRate = CreateSimpleQuoteUpsertRequest("USD/JPY", QuoteSeriesId.InstrumentIdTypeEnum.CurrencyPair, (150 + days), "USD", date);
-                    var inverseFxRate = CreateSimpleQuoteUpsertRequest("JPY/USD", QuoteSeriesId.InstrumentIdTypeEnum.CurrencyPair, 1m / (150 + days), "USD", date);
+                    var fxRate = BuildSimpleQuoteUpsertRequest("USD/JPY", QuoteSeriesId.InstrumentIdTypeEnum.CurrencyPair, (150 + days), "USD", date);
+                    var inverseFxRate = BuildSimpleQuoteUpsertRequest("JPY/USD", QuoteSeriesId.InstrumentIdTypeEnum.CurrencyPair, 1m / (150 + days), "USD", date);
 
                     upsertQuoteRequests.Add($"day_{days}_fx_rate", fxRate);
                     upsertQuoteRequests.Add($"day_{days}_inverseFx_rate", inverseFxRate);
                 }
             }
 
-            // CHECK quotes upsert was successful for all the quotes
-            var upsertResponse = _quotesApi.UpsertQuotes(scope, upsertQuoteRequests);
-            Assert.That(upsertResponse.Failed.Count, Is.EqualTo(0));
-            Assert.That(upsertResponse.Values.Count, Is.EqualTo(upsertQuoteRequests.Count));
+            return upsertQuoteRequests;
         }
-        
+
         /// <summary>
         /// This method upserts the reset quotes for the floating leg of a swap
         /// </summary>
-        private void UpsertResetQuotes(string scope, DateTimeOffset effectiveAt)
+        public static Dictionary<string, UpsertQuoteRequest> BuildResetQuotesRequest(string scope, DateTimeOffset effectiveAt)
         {
             // CREATE reset quotes in the desired date range
             var upsertQuoteRequests = new Dictionary<string, UpsertQuoteRequest>();
-            
-            var resetQuote = CreateSimpleQuoteUpsertRequest("BP00", QuoteSeriesId.InstrumentIdTypeEnum.RIC, 150, "USD", effectiveAt);
+
+            var resetQuote = BuildSimpleQuoteUpsertRequest("BP00", QuoteSeriesId.InstrumentIdTypeEnum.RIC, 150, "USD", effectiveAt);
             upsertQuoteRequests.Add($"resetQuote", resetQuote);
 
-            // CHECK quotes upsert was successful for all the quotes
-            var upsertResponse = _quotesApi.UpsertQuotes(scope, upsertQuoteRequests);
-            Assert.That(upsertResponse.Failed.Count, Is.EqualTo(0));
-            Assert.That(upsertResponse.Values.Count, Is.EqualTo(upsertQuoteRequests.Count));
+            return upsertQuoteRequests;
         }
 
-        private void UpsertEquityQuote(
+        public static Dictionary<string, UpsertQuoteRequest> BuildEquityQuoteRequest(
```

Comment:
Consider adding the /// <summary> docstrings like the other methods have

Reply:
Have added some more comments